app/src/main/res/values/strings.xml

```diff
@@ -39,7 +39,8 @@
   <string name="status_send_success">Send successfully</string>
   <string name="status_send_fail">Send failed</string>
   <string name="status_location_update">Location update</string>
-  <string name="status_connectivity_change">Connectivity change</string>
+  <string name="status_connectivity_change_online">Connectivity change: Online</string>
```

Comment:
Should we just rename these two to something like "status_network_online" (key) and "Network online" (value)? I think it will make it more readable.